tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -6067,6 +6083,13 @@ node_modules/chrome-devtools-frontend/front_end/extensions/ExtensionAPI.js(429,1
 node_modules/chrome-devtools-frontend/front_end/extensions/ExtensionAPI.js(529,12): error TS2339: Property '_fire' does not exist on type 'EventSinkImpl'.
 node_modules/chrome-devtools-frontend/front_end/extensions/ExtensionAPI.js(544,12): error TS2339: Property '_fire' does not exist on type 'EventSinkImpl'.
 node_modules/chrome-devtools-frontend/front_end/extensions/ExtensionAPI.js(551,12): error TS2339: Property '_fire' does not exist on type 'EventSinkImpl'.
+node_modules/chrome-devtools-frontend/front_end/extensions/ExtensionAPI.js(768,19): error TS2339: Property 'inspectedWindow' does not exist on type '{}'.
```

Comment:
better types, probably defineProperty?

Reply:
yep, we don't treat defineProperty as an expando assignment, so 

```js
var chrome = {}
chrome.devtools = {}
chrome.devtools.inspectedWindow = {}
```

works, but 

```js
var chrome = {}
Object.defineProperty(chrome, 'devtools', { value: {}, enumerable: true })
chrome.devtools.inspectedWindow = {}
```
 
does not. Unfortunately, I haven't got a repro that matches the error in chrome; the previous example gives me `chrome.devtools: any` in a separate buffer, not `chrome.devtools: {}` as I see on the new error line here.

Reply:
Filed #28049 to track this.